screen2.0/src/app/_mainsearch/GenomicRegion.tsx

```diff
@@ -109,7 +105,7 @@ const GenomicRegion = (props: { assembly: "mm10" | "GRCh38", header?: boolean })
                 }}
                 onKeyDown={(event) => {
                   if (event.code === "Enter") {
-                    handleSubmit()
+                    window.open(handleSubmit(), "_self")
                   }
                   if (event.code === "Tab" && !start) {
                     setStart("5205263")
```

Comment:
can we use constant variable for start and end here ?

Reply:
Yes, although the logic contained in handleSubmit() is needed for making the search icon work due to there being two input modes. For ease of maintenance I just called handleSubmit() when needed even if it could have been hard coded